src/components/reusable/tag/tag.ts

```diff
@@ -56,6 +56,12 @@ export class Tag extends LitElement {
   @property({ type: String })
   tagColor = 'spruce';
 
+  /**
+   * Determines if the tag is used in the notification component.
+   */
+  @property({ type: Boolean })
+  isNotification = false;
```

Comment:
We should use more generic property names like `description` instead, as components are isolated and do not relate to each other.